its_live_monitoring/src/sentinel2.py

```diff
@@ -35,23 +42,55 @@ def raise_for_missing_in_google_cloud(scene_name: str) -> None:  # noqa: D103
     response.raise_for_status()
 
 
+def add_data_coverage_to_item(item: pystac.Item) -> pystac.Item:  # noqa: D103
```

Comment:
```suggestion
def add_data_coverage_to_item(item: pystac.Item) -> pystac.Item:
```
Can remove the noqa since the function is documented now